install_python_tools.sh

```diff
@@ -106,6 +106,8 @@ function ensure_poetry_installed {
   echo "Installing Poetry."
   # Instructions verbatim from https://python-poetry.org/docs/
   curl --silent --show-error --location https://install.python-poetry.org | python3 -
+  # Use single quote intentionally to disable expansion.
+  echo 'export PATH="~/.local/bin:$PATH' >> ~/.bashrc
```

Comment:
@coverprice Can you take a look at this line, not sure if it is the correct way to add the `poetry` executable to `PATH`.

Reply:
Yeah, this line should probably be a standalone file added to `~/.bashrc.d`, i.e.

copy this into a new file like "configs/poetry_path.sh"
```shell
# Add poetry to the CLI path
export PATH="~/.local/bin:$PATH"
```

Replace the above line with:

```shell
mkdir -p ~/.bashrc.d
cp "${INSTALL_PYTHON_TOOLS_HERE}/configs/poetry.sh" ~/.bashrc.d/
```

Reply:
Also, I noticed the original has a bug, it's missing a double quote.

```
echo 'export PATH="~/.local/bin:$PATH' >> ~/.bashrc
```
Should be:
```
echo 'export PATH="~/.local/bin:$PATH"' >> ~/.bashrc
```

Reply:
Working on this...

And I've noticed `shellcheck` is complaining about the missing shebang in the scripts in the `.bashrc.d` folder. Should I add the shebang line?

<img width="909" alt="image" src="https://github.com/user-attachments/assets/d96d50cc-79cf-4941-a7c0-ac535b8701c9">

Reply:
There's no harm in adding one. Be sure to use `#!/usr/bin/env bash`.